docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>I think it might help to be little bit more verbose in the print statement to say:</p><p> </p><pre class="ql-syntax" spellcheck="false">print(f'Found best radius {np.round(rad, 2)} in time series {tsidx} starting at index location {ssidx}.')
</pre><p> </p><pre class="ql-syntax" spellcheck="false">Found best radius 0.87 in time series 4 starting at index location 1271.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>